plugins/modules/aws_az_info.py

```diff
@@ -70,8 +70,9 @@
     ]"
 '''
 
-from ansible_collections.amazon.aws.plugins.module_utils.aws.core import AnsibleAWSModule
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry, ansible_dict_to_boto3_filter_list, camel_dict_to_snake_dict
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
```

Comment:
@s-hertel, do you know the history here?  Why are we importing `camel_dict_to_snake_dict` and `snake_dict_to_camel_dict` into ec2.py and calling them from there, rather than from `dict_transformations`?

Reply:
@jillr They started out in ec2, but they proved more generally useful so they were moved to a common location (but existing modules weren't proactively corrected.)  Since I was going through these imports manually anyway, I removed the indirection.

Reply:
@flowerysong Yeah, that's right. +1 for fixing

Reply:
We should probably add a sanity test for this...